plugins/modules/azure_rm_keyvaultsecret_info.py

```diff
@@ -270,7 +270,7 @@ def exec_module(self, **kwargs):
                 if self.version == 'all':
                     self.results['secrets'] = self.get_secret_versions()
                 else:
-                    self.results['secrets'] = self.get_secret()
+                    self.results['secrets'] = [self.get_secret(self.name)]
```

Comment:
Won't this result in an array looking like `[None]` on a ResourceNotFound?

Reply:
Already update!